[2024-11-28T06:57:02.654+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-28T06:57:02.687+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_green_tripdata.dbt_transform.fct_trips.test scheduled__2024-11-27T00:00:00+00:00 [queued]>
[2024-11-28T06:57:02.704+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_green_tripdata.dbt_transform.fct_trips.test scheduled__2024-11-27T00:00:00+00:00 [queued]>
[2024-11-28T06:57:02.705+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-28T06:57:02.737+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): dbt_transform.fct_trips.test> on 2024-11-27 00:00:00+00:00
[2024-11-28T06:57:02.750+0000] {standard_task_runner.py:64} INFO - Started process 974 to run task
[2024-11-28T06:57:02.754+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nyc_green_tripdata', 'dbt_transform.fct_trips.test', 'scheduled__2024-11-27T00:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/dag_green_tripdata.py', '--cfg-path', '/tmp/tmpomkmania']
[2024-11-28T06:57:02.759+0000] {standard_task_runner.py:91} INFO - Job 102: Subtask dbt_transform.fct_trips.test
[2024-11-28T06:57:02.836+0000] {task_command.py:426} INFO - Running <TaskInstance: nyc_green_tripdata.dbt_transform.fct_trips.test scheduled__2024-11-27T00:00:00+00:00 [running]> on host 39c047f38cb8
[2024-11-28T06:57:02.975+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='muhii.smta@gmail.com' AIRFLOW_CTX_DAG_OWNER='Muhii' AIRFLOW_CTX_DAG_ID='nyc_green_tripdata' AIRFLOW_CTX_TASK_ID='dbt_transform.fct_trips.test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-27T00:00:00+00:00'
[2024-11-28T06:57:02.978+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-28T06:57:07.819+0000] {local.py:216} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2024-11-28T06:57:07.821+0000] {local.py:432} INFO - Cloning project to writable temp directory /tmp/tmp96lv_l4t from /opt/airflow/include/dbt
[2024-11-28T06:57:07.832+0000] {local.py:443} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/nyc_green_tripdata__dbt_transform/target/partial_parse.msgpack
[2024-11-28T06:57:07.851+0000] {config.py:332} INFO - Using user-supplied profiles.yml at /opt/airflow/include/dbt/profiles.yml
[2024-11-28T06:57:07.852+0000] {local.py:404} INFO - Trying to run dbtRunner with:
 ['test', '--models', 'fct_trips', '--project-dir', '/tmp/tmp96lv_l4t', '--profiles-dir', '/opt/***/include/dbt', '--profile', 'nyc_taxi', '--target', 'dev']
 in /tmp/tmp96lv_l4t
[2024-11-28T06:57:08.037+0000] {logging_mixin.py:188} INFO - 06:57:08  Running with dbt=1.8.7
[2024-11-28T06:57:09.334+0000] {logging_mixin.py:188} INFO - 06:57:09  Registered adapter: bigquery=1.8.3
[2024-11-28T06:57:14.188+0000] {logging_mixin.py:188} INFO - 06:57:14  Found 7 models, 34 data tests, 2 sources, 600 macros
[2024-11-28T06:57:14.195+0000] {logging_mixin.py:188} INFO - 06:57:14
[2024-11-28T06:57:14.882+0000] {logging_mixin.py:188} INFO - 06:57:14  Concurrency: 1 threads (target='dev')
[2024-11-28T06:57:14.885+0000] {logging_mixin.py:188} INFO - 06:57:14
[2024-11-28T06:57:14.892+0000] {logging_mixin.py:188} INFO - 06:57:14  1 of 12 START test not_null_fct_trips_do_location_id ........................... [RUN]
[2024-11-28T06:57:17.479+0000] {logging_mixin.py:188} INFO - 06:57:17  1 of 12 PASS not_null_fct_trips_do_location_id ................................. [PASS in 2.58s]
[2024-11-28T06:57:17.487+0000] {logging_mixin.py:188} INFO - 06:57:17  2 of 12 START test not_null_fct_trips_dropoff_time_id .......................... [RUN]
[2024-11-28T06:57:19.758+0000] {logging_mixin.py:188} INFO - 06:57:19  2 of 12 PASS not_null_fct_trips_dropoff_time_id ................................ [PASS in 2.27s]
[2024-11-28T06:57:19.769+0000] {logging_mixin.py:188} INFO - 06:57:19  3 of 12 START test not_null_fct_trips_extra .................................... [RUN]
[2024-11-28T06:57:21.775+0000] {logging_mixin.py:188} INFO - 06:57:21  3 of 12 PASS not_null_fct_trips_extra .......................................... [PASS in 2.00s]
[2024-11-28T06:57:21.783+0000] {logging_mixin.py:188} INFO - 06:57:21  4 of 12 START test not_null_fct_trips_improvement_surcharge .................... [RUN]
[2024-11-28T06:57:24.216+0000] {logging_mixin.py:188} INFO - 06:57:24  4 of 12 PASS not_null_fct_trips_improvement_surcharge .......................... [PASS in 2.43s]
[2024-11-28T06:57:24.224+0000] {logging_mixin.py:188} INFO - 06:57:24  5 of 12 START test not_null_fct_trips_passenger_count .......................... [RUN]
[2024-11-28T06:57:26.095+0000] {logging_mixin.py:188} INFO - 06:57:26  5 of 12 PASS not_null_fct_trips_passenger_count ................................ [PASS in 1.87s]
[2024-11-28T06:57:26.103+0000] {logging_mixin.py:188} INFO - 06:57:26  6 of 12 START test not_null_fct_trips_payment_type ............................. [RUN]
[2024-11-28T06:57:28.028+0000] {logging_mixin.py:188} INFO - 06:57:28  6 of 12 PASS not_null_fct_trips_payment_type ................................... [PASS in 1.92s]
[2024-11-28T06:57:28.038+0000] {logging_mixin.py:188} INFO - 06:57:28  7 of 12 START test not_null_fct_trips_pickup_time_id ........................... [RUN]
[2024-11-28T06:57:30.154+0000] {logging_mixin.py:188} INFO - 06:57:30  7 of 12 PASS not_null_fct_trips_pickup_time_id ................................. [PASS in 2.11s]
[2024-11-28T06:57:30.163+0000] {logging_mixin.py:188} INFO - 06:57:30  8 of 12 START test not_null_fct_trips_pu_location_id ........................... [RUN]
[2024-11-28T06:57:32.243+0000] {logging_mixin.py:188} INFO - 06:57:32  8 of 12 PASS not_null_fct_trips_pu_location_id ................................. [PASS in 2.08s]
[2024-11-28T06:57:32.256+0000] {logging_mixin.py:188} INFO - 06:57:32  9 of 12 START test not_null_fct_trips_rate_code_id ............................. [RUN]
[2024-11-28T06:57:34.211+0000] {logging_mixin.py:188} INFO - 06:57:34  9 of 12 PASS not_null_fct_trips_rate_code_id ................................... [PASS in 1.95s]
[2024-11-28T06:57:34.219+0000] {logging_mixin.py:188} INFO - 06:57:34  10 of 12 START test not_null_fct_trips_trip_distance ........................... [RUN]
[2024-11-28T06:57:37.857+0000] {logging_mixin.py:188} INFO - 06:57:37  10 of 12 PASS not_null_fct_trips_trip_distance ................................. [PASS in 3.64s]
[2024-11-28T06:57:37.867+0000] {logging_mixin.py:188} INFO - 06:57:37  11 of 12 START test not_null_fct_trips_vendor_id ............................... [RUN]
[2024-11-28T06:57:39.833+0000] {logging_mixin.py:188} INFO - 06:57:39  11 of 12 PASS not_null_fct_trips_vendor_id ..................................... [PASS in 1.96s]
[2024-11-28T06:57:39.845+0000] {logging_mixin.py:188} INFO - 06:57:39  12 of 12 START test unique_fct_trips_trip_id ................................... [RUN]
[2024-11-28T06:57:42.094+0000] {logging_mixin.py:188} INFO - 06:57:42  12 of 12 FAIL 59326 unique_fct_trips_trip_id ................................... [FAIL 59326 in 2.25s]
[2024-11-28T06:57:42.105+0000] {logging_mixin.py:188} INFO - 06:57:42
[2024-11-28T06:57:42.109+0000] {logging_mixin.py:188} INFO - 06:57:42  Finished running 12 data tests in 0 hours 0 minutes and 27.91 seconds (27.91s).
[2024-11-28T06:57:42.288+0000] {logging_mixin.py:188} INFO - 06:57:42
[2024-11-28T06:57:42.291+0000] {logging_mixin.py:188} INFO - 06:57:42  Completed with 1 error and 0 warnings:
[2024-11-28T06:57:42.293+0000] {logging_mixin.py:188} INFO - 06:57:42
[2024-11-28T06:57:42.296+0000] {logging_mixin.py:188} INFO - 06:57:42  Failure in test unique_fct_trips_trip_id (models/tests/schema.yml)
[2024-11-28T06:57:42.298+0000] {logging_mixin.py:188} INFO - 06:57:42    Got 59326 results, configured to fail if != 0
[2024-11-28T06:57:42.301+0000] {logging_mixin.py:188} INFO - 06:57:42
[2024-11-28T06:57:42.304+0000] {logging_mixin.py:188} INFO - 06:57:42    compiled code at target/compiled/nyc_taxi/models/tests/schema.yml/unique_fct_trips_trip_id.sql
[2024-11-28T06:57:42.306+0000] {logging_mixin.py:188} INFO - 06:57:42
[2024-11-28T06:57:42.309+0000] {logging_mixin.py:188} INFO - 06:57:42  Done. PASS=11 WARN=0 ERROR=1 SKIP=0 TOTAL=12
[2024-11-28T06:57:43.486+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-11-28T06:57:43.487+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-11-28T06:57:43.502+0000] {local.py:490} INFO - Inlets: [Dataset(uri='bigquery/northern-shield-437204-m0.nyc_taxi.fct_trips', extra=None)]
[2024-11-28T06:57:43.504+0000] {local.py:491} INFO - Outlets: []
[2024-11-28T06:57:43.505+0000] {local.py:590} INFO - Assigning inlets/outlets without DatasetAlias
[2024-11-28T06:57:43.506+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2024-11-28T06:57:43.554+0000] {dag.py:3947} INFO - Setting next_dagrun for nyc_green_tripdata to None, run_after=None
[2024-11-28T06:57:43.626+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-28T06:57:43.651+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/cosmos/operators/local.py", line 763, in execute
    result = self.build_and_run_cmd(context=context, cmd_flags=self.add_cmd_flags())
  File "/home/airflow/.local/lib/python3.9/site-packages/cosmos/operators/local.py", line 649, in build_and_run_cmd
    result = self.run_command(cmd=dbt_cmd, env=env, context=context)
  File "/home/airflow/.local/lib/python3.9/site-packages/cosmos/operators/local.py", line 502, in run_command
    self.handle_exception(result)
  File "/home/airflow/.local/lib/python3.9/site-packages/cosmos/operators/local.py", line 233, in handle_exception_dbt_runner
    raise AirflowException(f"dbt invocation completed with errors: {error_message}")
airflow.exceptions.AirflowException: dbt invocation completed with errors: unique_fct_trips_trip_id: Got 59326 results, configured to fail if != 0
[2024-11-28T06:57:43.658+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=nyc_green_tripdata, task_id=dbt_transform.fct_trips.test, run_id=scheduled__2024-11-27T00:00:00+00:00, execution_date=20241127T000000, start_date=20241128T065702, end_date=20241128T065743
[2024-11-28T06:57:43.681+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 102 for task dbt_transform.fct_trips.test (dbt invocation completed with errors: unique_fct_trips_trip_id: Got 59326 results, configured to fail if != 0; 974)
[2024-11-28T06:57:43.750+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-11-28T06:57:43.806+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-28T06:57:43.810+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
